alvr/dashboard/src/dashboard/components/settings_controls/presets/builtin_schema.rs

```diff
@@ -104,6 +104,39 @@ pub fn framerate_schema() -> PresetSchemaNode {
     })
 }
 
+pub fn encoder_quality_schema() -> PresetSchemaNode {
+    PresetSchemaNode::HigherOrderChoice(HigherOrderChoiceSchema {
+        name: "encoder_quality".into(),
+        strings: HashMap::new(),
+        flags: ["steamvr-restart".into()].into_iter().collect(),
+        options: [
+            ("Speed", "Speed", "P1"),
+            ("Balanced", "Balanced", "P3"),
+            ("Quality", "Quality", "P5"),
+        ]
+        .into_iter()
+        .map(|(key, val_amd, val_nv)| HigherOrderChoiceOption {
+            display_name: key.into(),
+            modifiers: [
+                string_modifier(
+                    "session_settings.video.advanced_codec_options.nvenc_overrides.nvenc_quality_preset",
```

Comment:
I haven't tested this, but I think you are missing a ".variant" at the end of the path. Same after amd_encoder_quality_preset below.